.github/scripts/release-samples.sh

```diff
@@ -37,6 +37,22 @@ echo "Channel version: ${CHANNEL_VERSION}"
 echo "Creating release branch for ${REPOSITORY}..."
 
 pushd $REPOSITORY
+
 git checkout -B "${CHANNEL_VERSION}"
+
+# Update bicepconfig.json br:biceptypes.azurecr.io/radius with the CHANNEL
+BICEPCONFIG_RADIUS_STRING_REPLACEMENT="br:biceptypes.azurecr.io/radius:${CHANNEL}"
+awk -v REPLACEMENT="${BICEPCONFIG_RADIUS_STRING_REPLACEMENT}" '{gsub(/br:biceptypes\.azurecr\.io\/radius:latest/, REPLACEMENT); print}' bicepconfig.json > bicepconfig_updated.json
```

Comment:
this is what we do for docs as well: https://github.com/radius-project/docs/blob/943c2cf3ea6b0d5cc24b06f6fa9de54e14207355/.github/scripts/release-docs.sh#L48